src/App.js

```diff
@@ -147,12 +167,20 @@ function App() {
   );
 
   const message = useMemo(() => {
-    if (!guesses.length) return "Guess the five letter word";
-    if (guesses.length > 0 && !done) return "You're making progress!";
-    if (done && giveup === false) return `Congratulations! It took you ${guesses.length} guesses.`;
-    if (done && giveup === true) return `You gave up after ${guesses.length-1} guesses.`;
+    const randomMessage = (messages) => messages[Math.floor(Math.random() * messages.length)];
+    if (!guesses.length) return randomMessage(messages.welcome);
+    if (!done) {
+    if (newMatches === 1) return randomMessage(messages.newMatched1);
```

Comment:
It looks like we are missing some indenting here. Most projects are very particular about indenting. For this project, each indent should be two spaces. I think lines 173-178 all need two more spaces at the front.

Reply:
See latest commit